indico_jacow/templates/mailing_lists.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'users/base.html' %}
+
+{% block user_content %}
+    <div id="mailing-lists"></div>
+    <script>
+        setupMailingList(document.querySelector('#mailing-lists'), {{ mailing_lists | tojson }});
+    </script>
+{% endblock %}
```

Comment:
Please configure your editor to always add a trailing linebreak. In VSCode that's the `files.insertFinalNewline` setting which should be enabled.